src/main/java/racingcar/view/InputValues.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.constant.SystemMessage;
+
+public class InputValues {
+
+    public String getName() {
+        System.out.println(SystemMessage.SET_CAR_NAME_MESSAGE.getMessage());
+        String getName =  Console.readLine();
+        Console.close();
+        return getName;
+    }
+
+    public String getCount() {
+        System.out.println(SystemMessage.SET_TRY_COUNT_MESSAGE.getMessage());
+        String getCount = Console.readLine();
+        Console.close();
```

Comment:
자동차 이름을 입력받는 getName() 메소드를 호출한 다음에 getCount() 메소드를 호출하면 결과적으로 Console.close() 메소드를 두 번 호출하게 되는 문제가 발생합니다. 만약 입력에 관해 문제가 생기셨던 것이라면 Console.close()를 프로그램 실행의 맨 마지막에만 1회 사용하도록 코드를 수정해 보시면 좋을 것 같습니다!

Reply:
너무나 감사합니다 ㅜㅜㅜ 적용해보겠습니다

Reply:
Cannot read the array length because "this.addValueService.carNameList" is null
at racingcar.service.FacadeService.setAllPositions(FacadeService.java:62)
	at racingcar.service.FacadeService.<init>(FacadeService.java:13)
	at racingcar.controller.CarRaceGameController.<init>(CarRaceGameController.java:12)
	at racingcar.Application.main(Application.java:8)


라고 합니다. 이 null에러를 어떻게 해결해야하는지 감이 안옵니다..

Reply:
에러 코드 맨 윗줄을 대략적으로 해석해보면 1차적으로 FacadeService 클래스의 메소드인 setAllPosition() 메소드에서 addValueService 클래스에 있는 String 배열인 carNameList의 길이를 모종의 이유로 읽어오지 못해서 발생하는 문제인 것 같습니다. 거기서부터 출발해보시면 좋을 것 같아요.